src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,84 @@
+.footer {
+  background-color: $color-dark;
+  grid-area: footer;
+
+  &__container {
+    padding-bottom: 80px;
+    @include padding-block;
+
+    @include onTablet {
+      display: flex;
+      flex-direction: column;
+    }
+
+    @include onSmallDesktop {
+      flex-direction: row;
+      justify-content: space-between;
+    }
+  }
+
+  &__menu {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+
+  &__logo {
+    width: 46px;
+    height: 28px;
+    margin-bottom: 48px;
+    @include hover (transform, scale(1.2));
+
+    @include onTablet {
+      display: block;
+      margin-bottom: 24px;
+    }
+  }
+
+  &__list {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    list-style: none;
+    padding: 0;
+
+    &:hover::after {
+      transform: scale(1);
+    }
+
+    @include onTablet {
+      flex-direction: row;
+      gap: 40px;
+    }
+  }
+
+  &__link {
```

Comment:
Same for the link. I have noticed that footer links does not have proper transition, and I found out the typo bug. If you would have link styles in one place (create link block) then this would not happen. Please extract link styles to **link** block. Check DRY principle (Don't repeate yourself) https://en.wikipedia.org/wiki/Don%27t_repeat_yourself